ci/rpm7/Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM centos:centos7
+
+ADD mongodb.repo /etc/yum.repos.d/
+ADD build.sh /opt/bin/
+ADD makefile /opt/archive/
+
+RUN ln -s /opt/bin/build.sh /usr/local/bin/build
+
+RUN yum -y install epel-release \
```

Comment:
What about PRs that would need to add new dependencies? For instance, let's consider this one https://github.com/telefonicaid/fiware-orion/pull/3194 that needs to install mosquitto libraries to make the tests to work. I understand the procedure in this case should be the following:

* Create PR A which needs new dependency. Travis fails in PR A
* Create PR B limited to modifications in this Dockerfile to include the new dependency. Travis will be ok in PR B (as actually hasn't include any new test)
* Merge PR B, so ci/rpm7/Dockerfile get updated in master
* Wait for next day :) in oder fiware/orion-ci:rpm7 being rebuilt from master, with the new dependency
* Re-run Travis on PR A, that now should work

Is this the way is supposed it would work? Is my understanding correct?

Reply:
Good question) It can be done simply, +1 step to check if orion-ci-rpm7 changed, and if true, rebuild image with tag. When pr will be landed, travis will rebuild main one image.

Reply:
It's a normal logic to have build artifacts in intermediate steps, and docker build itself have stages onboard.

Reply:
No problem if that is the behaviour but it should be documented in the README.md file (in a new subsection). You can more or less copy-paste from my sequence if you want.

This is important to avoid user wondering about "what my PR fails in travis if everything is ok?". With this piece of documentation the user learn that first a PR with the changes in ci/rpm7/Dockerfile has to be merged.

Reply:
I will try to add logic, that will do it automatically.

Reply:
Let make separate PR for Source and CI for now. In case https://github.com/telefonicaid/fiware-orion/pull/3194 mqtt dependencies added to the image.

Reply:
> Let make separate PR for Source and CI for now

Not sure what do you mean... If that means that each time a new dependency is needed a PR has to be done on the ci/rpm7/Dockerfile before the one using that depenency in order the CI-travis system to work, that's fine. But please explain it in the README.md. You can use a copy-paste of my bullet-list in the first comment in this thread if you find it useful.

Reply:
Fixed in https://github.com/caa06d9c/fiware-orion/pull/7/commits/3b1c5c13beb79e35486dbef0115e438242fc6de2